README.md

```diff
@@ -159,3 +159,22 @@ console.assert(foo.toString.includes("...")); // oops
 This basically makes the hiding toothless, so that you do not gain the encapsulation or memory usage benefits.
 
 You could try to patch around it by saying that only setting it to `true` works, and setting it to `false` or deleting the property does nothing. But that's just strange. We already have a mechanism for changing the state of an object in a non-reversible way: call a method on it. Thus, the `foo.hide()` proposal above (which has its own problems).
+
+### How does this affect devtools or other methods of inspecting a function's implementation details?
+
+This proposal, like all JavaScript language proposals, only impacts the behavior of JavaScript code. Developer tools are outside the scope, and this proposal does not intend to change how they behave in any way. So, functions with their implementation hidden via the proposal's directive can still be inspected by devtools.
+
+Concretely, this proposal only impacts two things:
+
+* The JavaScript string values produced by `Function.prototype.toString()`
+* The values produced by not-yet-standardized properties like `Error.prototype.stack` or other parts of the [error stacks proposal](https://github.com/tc39/proposal-error-stacks).
+
+The proposed specification text does _not_ impact any of the following:
+
+* What the developer sees on their screen when they do `console.log(someFunction)`.
+* What the developer sees on their screen when they look at the source code or network-response panels of devtools.
+* What error stack traces the developer sees on their screen when an unhandled exception or rejection occurs.
```

Comment:
this bullet point seems to contradict the "only impacts two things" list?

Reply:
It does not impact what you see on your console. It does impact the string returned by `.stack`. Does that make sense? Any ideas on how to make it clearer?

Reply:
not sure, perhaps?
```suggestion
* What error stack traces the developer eventually sees on their screen when an unhandled exception or rejection occurs (given that engines can choose to decorate or transform the `.stack` string any way they like)
```

Reply:
Yeah, something like that, although I'd phrase it more like "given that this doesn't have to be related to the JavaScript-exposed .stack string". Thanks, will push an update shortly.